via/static/scripts/video_player/components/test/VideoPlayerApp-test.js

```diff
@@ -69,6 +72,54 @@ describe('VideoPlayerApp', () => {
     $imports.$restore();
   });
 
+  describe('app layout', () => {
+    context('multi-column layout for wider widths', () => {
+      for (const appSize of ['md', 'lg', 'xl', '2xl']) {
+        it('renders a top bar with branding and filter input', () => {
+          fakeUseAppLayout.returns(appSize);
+          const wrapper = createVideoPlayer();
+
+          const topBar = wrapper.find('[data-testid="top-bar"]');
+          assert.isTrue(topBar.exists());
+          assert.isTrue(
+            topBar.find('[data-testid="hypothesis-logo"]').exists()
+          );
+          assert.isTrue(topBar.find('[data-testid="filter-input"]').exists());
```

Comment:
IIRC `wrapper.exists(<selector>)` also works as a shorthand for `wrapper.find(<selector>).exists()`.